pkg/detector/ospkg/echo/echo_test.go

```diff
@@ -0,0 +1,146 @@
+package echo
+
+import (
+	"sort"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/aquasecurity/trivy-db/pkg/db"
+	dbTypes "github.com/aquasecurity/trivy-db/pkg/types"
+	"github.com/aquasecurity/trivy/internal/dbtest"
+	ftypes "github.com/aquasecurity/trivy/pkg/fanal/types"
+	"github.com/aquasecurity/trivy/pkg/types"
+)
+
+func TestScanner_Detect(t *testing.T) {
+	type args struct {
+		pkgs   []ftypes.Package
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    []types.DetectedVulnerability
+		wantErr string
+		fixtures []string
+	}{
+		{
+			name: "happy path - detect vulnerabilities",
+			fixtures: []string{
+				"testdata/fixtures/echo.yaml",
+				"testdata/fixtures/data-source.yaml",
+			},
+			args: args{
+				pkgs: []ftypes.Package{
+					{ID: "echo", Version: "1.0.0"},
+					{ID: "python3", SrcName: "python3", Version: "3.6.8"},
+					{ID: "apache2", SrcName: "apache2", Version: "2.4.24"},
```

Comment:
very strange fields.
Can you update fields as for `dpkg`?

Reply:
Done 👍